File: examples/ToolRAG/tool-rag/prompt.ts
import braintrust from "braintrust";
import { PROJECT_NAME } from "./constants";
import { retrieval } from "./retrieval";

const project = braintrust.projects.create({
  name: PROJECT_NAME,
});

export const prompt = project.prompts.create({
  name: "Doc Search",
  messages: [
    {
      role: "system",
      content:
        "You are a helpful assistant that can " +
        "answer questions about the Braintrust documentation.",
    },
    {
      role: "user",
      content: "{{{question}}}",
    },
  ],
  model: "gpt-4o-mini",
  tools: [retrieval],
  ifExists: "replace",
});

File: examples/ToolOCR/tool-ocr/prompt.py
import braintrust
import ocr


project = braintrust.projects.create(name="toolOCR")


prompt = project.prompts.create(
    name= "Recipe text generator",
    messages= [{"role": "system", "content": "You are a helpful assistant that ",},
        {"role": "user", "content": "{{{question}}}",},
    ],
model= "gpt-4o-mini",
tools= [ocr],
ifExists= "replace",
)